lib/util/index.js

```diff
@@ -1,10 +1,7 @@
 "use strict";
 
+let fs = require("fs");
 let path = require("path");
```

Comment:
Might as well do this while we're at it:
```suggestion
let fs = require("node:fs");
let path = require("node:path");
```